src/components/Carousel.js

```diff
@@ -603,9 +603,17 @@ export default class Carousel extends Component {
    * @param {string} name of an element
    * @return {ReactElement} element with added handler
    */
-  renderArrowWithAddedHandler = (element, onClick, name) => (
+  renderArrowWithAddedHandler = (element, onClick, name, disable = false) => (
```

Comment:
Thanks for this PR.

Could you add the new props to README and docs/ demo?

Reply:
Should i add it to the custom arrows file?

Reply:
yes, please

Reply:
@piotr-s-brainhub updated the docs as well. Please verify and let me know.
Thanks!

Reply:
@piotr-s-brainhub why do we have two functions for `renderRightArrow` and `renderLeftArrow`. How about combining them into a single function `renderArrow` ?

Reply:
@udaypydi 

I cannot find `renderRightArrow` or `renderLeftArrow`

Reply:
you mean `renderArrowLeft` and `renderArrowRight`?

Reply:
IMO there are enough differences to have two separate functions.
It's better to have a bit too redundant code than a bit too unified because the former causes code duplications which isn't good but manageable but the latter makes the code too complex and less readable because of multiple conditions.

Reply:
yeah got it completely.
Thanks!